### PR TITLE
Replace broken media in examples with some that actually work

### PR DESCRIPTION
**Pull request by mfairchild365**

### CODE REVIEW

**Comment by kabel** [comment]
Whomp whomp. Looks like the tests might have some weird file ordering that broke.

**Comment by kabel** [comment]
Because we have two videos in the video example, I'd like to keep _one_ that doesn't load so we can more effectively test the "failed" state.

**Comment by mfairchild365** [comment]
Good point.

File: wdn/templates_4.1/examples/audioplayer.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 </head>
 <body data-version="4.1">
     <div id="example-code">
-<audio class='wdn_player' preload="auto" src="http://elkhorn.unl.edu/citmp3/current/ogg2.mp3"></audio>
+<audio class='wdn_player' preload="auto" src="http://mediahub.unl.edu/uploads/e69c5690123db31675e38132303fd3ff.mp3"></audio>
```
Comment (kabel) [inline wdn/templates_4.1/examples/audioplayer.html]: Do me a favor and fix these quotes.

File: wdn/templates_4.1/examples/audioplayer.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
...
 <script type="text/javascript">
```
Comment (kabel) [inline wdn/templates_4.1/examples/audioplayer.html]: If you're feeling generous, also remove this script type for coding standards.

File: wdn/templates_4.1/examples/audioplayer.html (hunk 1 above)
Comment (mfairchild365) [inline wdn/templates_4.1/examples/audioplayer.html]: make me


 
.
 
.
  
(okay)

. 

:smile:

**Comment by kabel** [comment]
Oh man. You made me merge extra whitespace. :(

File: tests/Accessibility/accessibility.phpt [hunk 3]
```diff
@@ -32,6 +32,9 @@ class AccessibilityTester {
             $files_to_check[] = $file_info->getFilename();
         }
 
+        //Make sure that files are ordered by file name
+        sort($files_to_check);
+
```
Comment (kabel) [inline tests/Accessibility/accessibility.phpt]: Trailing whitespace !?
I'll fix it.

**Pull request merged**